grpc/rust/BUILD

```diff
@@ -0,0 +1,64 @@
+load("@rules_rust//proto:proto.bzl", "rust_grpc_library")
+load("@rules_rust//proto:toolchain.bzl", "rust_proto_toolchain")
+load("@rules_rust//rust:defs.bzl", "rust_binary")
+
+# TODO: once we clarify how we should compile the gRPC + Protobuf library, delete this target
+rust_binary(
```

Comment:
This was my second attempt at getting a working gRPC + Protobuf Rust compiler binary.

It works, but, ideally we would like to not write our own `main.rs`, but rather delegate it to an established Bazel Rust rules library - ie: `rules_rust` or other.